_posts/pl/2021-02-01-wszystko-co-chcielibyscie-wiedziec-o-fontach-ale-baliscie-sie-zapytac.md

```diff
@@ -0,0 +1,104 @@
+---
+layout:    post
+title:     Wszystko, co chcielibyście wiedzieć o fontach, ale baliście się zapytać
+published: false
+date:      2021-01-01 09:00:00 +0100
+author:    pgrobelny
+lang:      pl
+image:     
+tags:
+    - font
+    - czcionka
+    - rem
+    - woff
+    - font-face
+---
+
+Większość osób, która pracowała na frontendzie na pewno miała do czynienia z fontami, czy to wrzucała je do projektu,
```

Comment:
Większość osób, które pracowało[...]

Reply:
"większość" to ona ~Chrapek mi o tym napisał i tak poprawiłem

Reply:
No dobra, ale wg mnie wszystkie osoby pracowały, a nie większość. Zostawmy jak jest.

Reply:
Chociaż nie, wróć. ~Chrapek dobrze Ci napisał - "miała do czynienia" i "zmieniała sposób", nie pisał żeby zmieniać "pracowało".

Reply:
ale jak
"Większość osób, które pracowało na frontendzie na pewno miała do czynienia"? dziwnie to brzmi